### PR TITLE
Resolve description lint error

### PR DESCRIPTION
**Pull request by Szarny**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by nghialv — APPROVED** [review]
Welcome! 👍

File: web/src/components/application-detail-page/application-detail/index.tsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ export const ApplicationDetail: FC<ApplicationDetailProps> = memo(
 
     const piped = useAppSelector(selectPipedById(app?.pipedId));
     const isSyncing = useIsSyncingApplication(app?.id);
-    const description = app?.description.replace(/\\\n/g, "  \n");
+    const description = app?.description.replace(/\\\n/g, "  \n") || '';
```
[the remark this answers is not in the document]
Comment (Szarny) [reply web/src/components/application-detail-page/application-detail/index.tsx]: I fixed it!

**Comment by khanhtc1202** [comment]
@Szarny plz rebase master to fix this CI failed

**Comment by Szarny** [comment]
I confirmed the labeler workflow works correctly ✅

**Review by khanhtc1202 — APPROVED** [review]
🙌

**Comment by khanhtc1202** [comment]
@Szarny approved, feel free to merge this when you want 👌

**Pull request merged**